examples/vpc-and-vpn/main.tf

```
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+
+terraform {
+  required_version = "0.15.4"
```

maybe we can start to relax the constraints here for future-proofing

```suggestion
  required_version = ">=0.15.4"
```